VAMobile/documentation/development/FrontEnd/DeepLinks.md

```diff
@@ -0,0 +1,294 @@
+---
+title: Universal and app links
+---
+
+## Overview
+
+Universal links (iOS) and App links (Android), a type of deep linking, enable an app to respond to specific URLs by navigating to a particular screen or triggering a defined action even if the app is not currently running.
+
+These type of deep linking function across both web and mobile platforms. For example, when a user taps a link, it can open the related screen directly within the React Native app instead of loading a webpage. This creates a more seamless and integrated user experience between mobile and web content.
+
+## Configurations
+:::note
+Deep linking support for both iOS and Android should already be configured. However, here's a brief overview of the setup process for those who want to review or customize the implementation.
+
+If you are adding a new deeplink then go **[adding a new linking path to app](./DeepLinks.md#adding-a-new-linking-path-in-the-app)** section.
+:::
+
+### iOS
+
+**An apple-app-site-association (AASA) file has been uploaded to our [website server](https://github.com/department-of-veterans-affairs/content-build/tree/main/src/site/assets/.well-known). New paths will require a PR to the [content-build repo](https://github.com/department-of-veterans-affairs/content-build).** 
```

Comment:
"our website server" is the va.gov site right? We should be specific where possible